src/common/typedPubsub.ts

```diff
@@ -139,11 +138,13 @@ export type PubSubSchema = {
     organizationId: Organization['id'];
     memberId: User['id'];
   };
-  'api.v1.brief-generate': UserBriefingRequest & {
+  'api.v1.brief-generate': {
+    payload: UserBriefingRequest;
```

Comment:
changing topic signature so we can extend easier, but its fine we are still not in prod